crates/host/Cargo.toml

```
@@ -18,13 +18,16 @@ rand = "0.8"
 bimap = "0.6"
 bytes = "1"
 hex = "0.4"
+thiserror = "1"
 
 [lib]
 name = "holochain_wasmer_host"
 crate-type = ["cdylib", "rlib"]
 path = "src/host.rs"
 
 [features]
-default = ["error_as_host"]
+default = ["error_as_host", "wasmer_sys"]
 debug_memory = []
 error_as_host = ["holochain_wasmer_common/error_as_host"]
+wasmer_sys = ["wasmer-middlewares", "wasmer/default"]
```

```suggestion
wasmer_sys = ["dep:wasmer-middlewares", "wasmer/default"]
```

I prefer this syntax. It's optional, but makes it clearer what's a feature and what is including a library. I think it also prevents IDEs and other tools from telling you that `wasmer-middlewares` is a feature

---

At some point, I hope Rust will make that syntax the default and required